weblate/billing/models.py

```diff
@@ -249,7 +256,9 @@ def all_projects(self):
 
     @cached_property
     def projects_display(self):
-        return ", ".join(str(x) for x in self.all_projects)
+        return format_html_join_comma(
+            "{}", (x for x in list_to_tuples(self.all_projects))
```

Comment:
```suggestion
            "{}", list_to_tuples(self.all_projects)
```

...and pretty much anywhere else.